public/assets/sass/carousel.scss

```diff
@@ -69,9 +69,9 @@
   border-radius: 50%;
   display: inline-block;
   transition: background-color 0.6s ease;
-}
 
-.active,
-.dot:hover {
-  background-color: #717171;
+  &.active,
```

Comment:
By moving `.active` into the `.dot` class, we prevent it from conflicting with any other uses of `.active` throughout the site, such as in the navbar